test/support/parse-input.test.js

```diff
@@ -7,7 +7,7 @@ const {
   getDomainIdentifierArg,
   getProjectIdentifierArg,
   readConfigFile,
-  reqType 
+  reqType, splitFlagParams
```

Comment:
To keep the format consistent - could you add it on a new line?

Reply:
Automatically imported it when I wrote the test 🤦  I'll change it now